src/content/configuration/other-options.md

```diff
@@ -143,33 +144,6 @@ module.exports = {
 };
 ```
 
-### `cache.loglevel`
```

Comment:
This isnt a way to go, it's a breaking change, therefore we add
```
W> Removed in webpack 5
```

Reply:
This feature is available in webpack 4 currently

Reply:
Got it, currently undocumented though? 👍 

I will fix!

Reply:
Just confirmed that it is not available in latest webpack, for some reason i was sure it was added alongside logging at 4.37.0, can you please re-use the first commit? I see there was a force push

Reply:
sorry for the confusion there, logging caught me off

Reply:
Updated 👌